client/js/libs/handlebars/localetime.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+const moment = require("moment");
+
 module.exports = function(time) {
-	return new Date(time).toLocaleString();
+	return moment(time).format("MMMM DD, YYYY – HH:mm:ss");
```

Comment:
Yeah, my comment in parens betrayed me. I never really noticed that 12/24h format difference before today.

Same comment as before though, I'd advocate for `DD MMMM YYYY, HH:mm:ss`.